soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -349,6 +398,13 @@ static int siwg917_nwp_init(void)
 		return -EINVAL;
 	}
 
+	/* Check if the NWP firmware version is correct */
+	status = siwx91x_check_nwp_version();
+	if (status < 0) {
+		LOG_ERR("NWP firmware version is incorrect");
```

Comment:
I think it'd be helpful to include the version string in the log output, so the developer can cross check what they actually have.

Reply:
Ah, nevermind, I see you have that in the `check_nwp_version()` function. However, I think this is relevant info *always*, i.e. I wouldn't put it behind `LOG_DBG()`.